webpack.config.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     path: path.join(__dirname, outputDirectory),
     filename: 'bundle.js'
   },
+  resolve: {
+    modules: [path.resolve('./node_modules'), path.resolve('./src/client')]
```

Comment:
do we need `path.resolve('./node_modules')` if a webpack explicitly is looking into node_modules folder with no need of providing a full relative path ??

Reply:
path.resolve('./node_modules') is necessary, without it, the webpack can't compile the app

Reply:
please try following:
modules: [path.resolve(__dirname, 'src/client'), 'node_modules']